### PR TITLE
Cart list

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Created cart list page along with functionalities like

1.  add to cart button which on click changes to got to cart
2. a remove button inside the cart
3. quantity counter of the items inside the cart
4. bill box that gives calculation of the price, discount and final total price

### CODE REVIEW

File: src/Pages/Cart List/Components/billBox.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import '../cart-list.css'
+import {useUserData} from '../../../Context/userDataContext'
+
+
+
+
```
Comment (manojasarsa) [inline src/Pages/Cart List/Components/billBox.jsx:6]: you can remove extra space

File: src/Pages/Cart List/cartListDataHandlers/cartListProductCounterHandler.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                }else{
+                    response = await cartListProductCounter(id, token, operation);
+            }
+            if (response.status === 200) {
+                toast.success("Updated your cart !");
+                userDataDispatch({type: "SET_CARTLIST", payload: response?.data.cart})
+            }
+            setFetchingCartList(false)
+        } catch (error) {
+            console.log(error)
```
Comment (manojasarsa) [inline src/Pages/Cart List/cartListDataHandlers/cartListProductCounterHandler.js:20]: can use `console.error` here

**Review by manojasarsa — APPROVED** [review]
Code looks clean. GTG!

**Review by ArjunJunna — APPROVED** [review]
looks good to go.

**Pull request merged**